vm-zone-move/README.md

```diff
@@ -0,0 +1,51 @@
+# Failover scripts
+
+During zone down scenario, the VM cannot be accessed. These scripts will bring back the vm in a different zone or region depending upon the type of disks (ZRS/LRS) attached to the vm. Affectively moving the VM. 
```

Comment:
During zone down scenario, customers will not be able to access impacted VMs and consequently the data persisted in their data disks.